apps/random/main.c

```diff
@@ -50,10 +34,31 @@ int main(void)
 	uint8_t rsp[CMDLEN_MAXBYTES];
 	uint8_t in;
 
```

Comment:
Why a newline?

Reply:
Think it increases readability to group them. But fixed in new repo.